templates/registration/login.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Login</h1>
+  {% if not user.is_authenticated %}
+    <form method="post">
+      {% csrf_token %}
+      {{ form.as_p }}
+      <button type="submit">Login</button>
+    </form>
+  {% else %}
+    <p><a href="{% url 'home' %}">Home page</a></p>
```

Comment:
The URL tag `{% url 'home' %}` may not resolve correctly if there is no URL pattern named 'home'. Ensure that the correct URL name is used, such as `{% url 'taxi:index' %}` if 'index' is the name of the home page view in the 'taxi' app.